### PR TITLE
Add temperature and voltage and SED HK and commanding

### PR DESCRIPTION
**Pull request by mbenson1**

### CODE REVIEW

File: apps/to/fsw/src/to_output_queue.c [hunk 1]
```diff
@@ -240,3 +489,23 @@ osalbool TO_OutputChannel_Query(uint16 ChannelIdx)
...
+{
+//    uint32 msgLen;
+//    uint32 textStart = offsetof(CFE_EVS_Packet_t, Payload) + offsetof(CFE_EVS_Packet_Payload_t, Message);
+//    uint32 textLength = strnlen(msgPtr->Payload.Message, CFE_EVS_MAX_MESSAGE_LENGTH);
+//
+//    msgLen = textStart + textLength + 1;
+//
+//    return msgLen;
+
+	return 20;
```
Comment (cnalley) [inline apps/to/fsw/src/to_output_queue.c]: Hardcoded 20?

File: apps/to/fsw/src/to_priority_queue.c [hunk 2]
```diff
@@ -60,10 +51,9 @@ void TO_PriorityQueue_ResetCountsAll(TO_ChannelData_t* channel)
 
     for (i = 0; i < TO_MAX_PRIORITY_QUEUES; ++i)
     {
-        channel->DumpTbl.PriorityQueue[i].DroppedMsgCnt = 0;
-        channel->DumpTbl.PriorityQueue[i].QueuedMsgCnt = 0;
-        channel->DumpTbl.PriorityQueue[i].HighwaterMark = 0;
-        channel->DumpTbl.PriorityQueue[i].CurrentlyQueuedCnt = 0;
+//        channel->DumpTbl.PriorityQueue[i].DroppedMsgCnt = 0;
```
Comment (cnalley) [inline apps/to/fsw/src/to_priority_queue.c:54]: Commented reset counts?

File: apps/to/fsw/src/to_priority_queue.c [hunk 3]
```diff
@@ -231,134 +199,168 @@ int32 TO_PriorityQueue_TeardownAll(TO_ChannelData_t *channel)
...
-    return status;
+    return CFE_SUCCESS;
 }
 
+/* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * */
+/*                                                                 */
+/* Push a message on a priority queue.                             */
+/*                                                                 */
+/* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * */
+//int32 TO_PriorityQueue_QueueMsg(
```
Comment (cnalley) [inline apps/to/fsw/src/to_priority_queue.c]: QueueMsg no longer used?

File: apps/to/fsw/src/to_output_queue.c (hunk 1 above)
Comment (mbenson1) [reply apps/to/fsw/src/to_output_queue.c]: Fixed.

File: apps/to/fsw/src/to_priority_queue.c (hunk 2 above)
Comment (mbenson1) [reply apps/to/fsw/src/to_priority_queue.c:54]: Submitted ticket on this. #65

File: apps/to/fsw/src/to_priority_queue.c (hunk 3 above)
Comment (mbenson1) [reply apps/to/fsw/src/to_priority_queue.c]: No.  I deleted this function.

**Review by cnalley — APPROVED** [review]

**Pull request merged**